File: tests/fixtures/golden/typescript/visibilityPrivateProber.ts
class Prober {
  // Calls SecretHelper.doSecret via an unimported class reference (no
  // `import { SecretHelper }`), so there's no import candidate and this
  // reaches the two-segment name fallback. Before #75, `SecretHelper.
  // doSecret` was the sole two-segment match in the whole fixture and
  // bound despite being private and declared in a different file. #75's
  // visibility guard must refuse it.
  callRemotely(): string {
    return SecretHelper.doSecret();
  }
}

File: tests/fixtures/golden/rust/src/visibility_private_prober.rs
/// Calls `secret_helper`, which exists only in `visibility_private_owner`
/// as a private (non-`pub`) function -- no `use` brings it into scope
/// here, so this is a bare, unqualified call. Before #75, the bare-name
/// fallback found exactly one same-language candidate in the whole fixture
/// (the only `secret_helper`) and bound to it despite it being private and
/// declared in a different file. #75's visibility guard must refuse that
/// cross-file bind and leave the call `UNRESOLVED(private)`.
pub fn call_remotely() -> &'static str {
    secret_helper()
}
